app-office/calcurse/patches/calcurse-4.3.0.patch

```diff
@@ -0,0 +1,50 @@
+From 01abcd76f624ef17b56d85c8135850b15d94a672 Mon Sep 17 00:00:00 2001
+From: Andy <andy@keyboardfire.com>
+Date: Sun, 3 Dec 2017 13:25:59 +0000
+Subject: Fix version and don't check for lm or pthread
+
+---
+ configure.ac | 20 +-------------------
+ 1 file changed, 1 insertion(+), 19 deletions(-)
+
+diff --git a/configure.ac b/configure.ac
+index b1480ec..d516199 100644
+--- a/configure.ac
++++ b/configure.ac
+@@ -3,7 +3,7 @@
+ #-------------------------------------------------------------------------------
+ AC_PREREQ(2.59)
+ AC_INIT([calcurse],
+-        m4_esyscmd([build-aux/git-version-gen .version]),
++        4.3.0,
+         [bugs@calcurse.org])
+ AM_INIT_AUTOMAKE
+ #m4_ifdef([AM_SILENT_RULES],[AM_SILENT_RULES([yes])])
+@@ -50,24 +50,6 @@ AC_CHECK_FUNC(initscr,,
+   fi
+   LIBS="$LIBS -l$available_ncurses"
+ ])
+-
+-AC_CHECK_HEADERS([pthread.h], [
+-    AC_CHECK_LIB(pthread, pthread_create, [
+-	LIBS="$LIBS -pthread"
+-	AC_DEFINE(HAVE_LIBPTHREAD, 1, 
+-		  [Define to 1 if you have the 'pthread' library (-pthread).])
+-    ], 
+-    AC_MSG_ERROR(The pthread library is required in order to build calcurse!))
+-], AC_MSG_ERROR(The pthread header is required in order to build calcurse!))
+-
+-AC_CHECK_HEADERS([math.h], [
+-    AC_CHECK_LIB(m, exp, [
+-	LIBS="$LIBS -lm"
+-	AC_DEFINE(HAVE_LIBMATH, 1, 
```

Comment:
Haiku have -lm

Reply:
Yep, we have a libm, even if it's just a *fake* lib.
(On Haiku, the "Math" lib is embedded in libroot.)
So you can remove the PATCHES line from the recipe (and delete the patch itself.)